### PR TITLE
take a break

### PR DESCRIPTION
**Pull request by 3zzazakl**
- rename file and get done word after merging with main branch
- adding a new line to steps file
- setting goals for today
- set notes

### CODE REVIEW

File: concepts/console.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    
+    def do_hello(self, args):
+        """_summary_
+
+        Args:
+            args (_type_): _description_
+        """
+        print("Hello AirBnB")
+    
+if __name__ == "__main__":
```
Comment (azzaz3kl) [inline concepts/console.py:23]: practice new tech

File: concepts/console.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    console = myConsole()
+    console.cmdloop()
```
Comment (azzaz3kl) [inline concepts/console.py:25]: new instance

File: concepts/person.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"name": "Azzaz", "age": 27}
```
Comment (azzaz3kl) [inline concepts/person.json:1]: done

File: concepts/test.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+
+# read json str from file:
+with open("person.json", "r") as f:
+    person_1_str = f.read()
+
+person_1_dict_format = json.loads(person_1_str)
+print(person_1_dict_format)
+
+
```
Comment (azzaz3kl) [inline concepts/test.py:38]: test dict and json

File: concepts/test.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+def create_inst_from_dict(className, data_dict):
+    inst = className.__new__(className)
+    for key, value in data_dict.items():
+        setattr(inst, key, value)
+    return inst
+
+person_instance = create_inst_from_dict(person, person_1_dict_format)
+print("Deserialized instance: ", person_instance)
```
Comment (azzaz3kl) [inline concepts/test.py:46]: yeah

**Review by azzaz3kl — APPROVED** [review]
Take a break

**Review by azzaz3kl — COMMENTED** [review]
take a break

**Review by 3zzazakl — COMMENTED** [review]
done

**Pull request merged**